pkg/conformance/deploy.go

```diff
@@ -116,6 +116,10 @@ func (r *TestRunner) Deploy(ctx context.Context, focus string, verboseGinkgo boo
 			Name:  "E2E_EXTRA_ARGS",
 			Value: strings.Join(r.config.ExtraArgs, " "),
 		},
+		{
+			Name:  "TZ",
+			Value: "UTC+",
+		},
```

Comment:
Did you mean to include TZ info for this PR? I know we have an open issue regarding setting TZ appropriatly.

Reply:
sorry i had those changes locally, and I mistakenly added those, I will remove this